File: Seminar/src/main/java/org/sopt/Seminar/domain/post/exception/PostNotFoundException.java
package org.sopt.Seminar.domain.post.exception;

import org.sopt.Seminar.global.common.exception.ErrorType.NotFound;
import org.sopt.Seminar.global.common.exception.SoptException;

public class PostNotFoundException extends SoptException.NotFoundException {
    public PostNotFoundException() {
        super(NotFound.POST_NOT_FOUND, "해당 게시물을 찾을 수 없습니다");
    }
}

File: kopring/week2/kotlin-semina/src/main/kotlin/org/sopt/kotlinsemina/domain/post/exception/PostNotFoundException.kt
package org.sopt.kotlinsemina.domain.post.exception

import org.sopt.kotlinsemina.global.common.exception.ErrorType.NotFound.*
import org.sopt.kotlinsemina.global.common.exception.SoptException

class PostNotFoundException: SoptException.NotFoundException(POST_NOT_FOUND, "해당 게시물이 존재하지 않습니다.")